2/2.go

```diff
@@ -85,13 +88,8 @@ func main() {
 	countTaskOne, countTaskTwo := 0, 0
 
 	for _, p := range passwords {
-		if p.isValidTaskOne() {
-			countTaskOne++
-		}
-
-		if p.isValidTaskTwo() {
-			countTaskTwo++
-		}
+		countTaskOne += p.isValidTaskOne()
+		countTaskTwo += p.isValidTaskTwo()
```

Comment:
Určitě bych zachoval původní podmínky, protože validační funkce by měly opravdu vracet spíš boolean hodnotu:

```go
countTaskOne, countTaskTwo := 0, 0
for _, p := range passwords {
	if p.isValidTaskOne() {
		countTaskOne++
	}
	if p.isValidTaskTwo() {
		countTaskTwo++
	}
}

fmt.Println("Task one:", countTaskOne)
fmt.Println("Task two:", countTaskTwo)
```

Reply:
Rozumím, navazuje to na boolean hodnoty